src/events/ready.js

```diff
@@ -1,19 +1,18 @@
 module.exports = {
   name: "ready",
   execute(bot) {
-    const userCount = bot.users.cache.filter((u) => !u.bot).size;
-    const channelCount = bot.channels.cache.size;
-    const interval = 1800000; /* 30 minutes */
-
+    const statuses = [
+      ` ${bot.guilds.cache.size} servers.`,
```

Comment:
Could you use variables here (for server count, user count and channel count)? eg `const serversCount = bot.guilds.cache.size`